docs/cloud/managing-airbyte-cloud/understand-airbyte-cloud-limits.md

```diff
@@ -3,6 +3,8 @@
 Understanding the following limitations will help you more effectively manage Airbyte Cloud.
 
 * Max number of workspaces per user: 5*
```

Comment:
Let's make this 3, instead of 5. No "internal" use-case should require more than one workspace for each of "production", "staging" and "dev" (total of 3 workspaces)

Reply:
Updated!